src/gantt.ts

```diff
@@ -2436,23 +2467,14 @@ export class Gantt implements IVisual {
      */
     private taskProgressRender(
         taskSelection: Selection<Task>): void {
-        let taskProgressShow: boolean = this.viewModel.settings.taskCompletion.show;
+        const taskProgressShow: boolean = this.viewModel.settings.taskCompletion.show;
 
-        let index = 0, groupedTaskIndex = 0;
         let taskProgress: Selection<any> = taskSelection
             .selectAll(Selectors.TaskProgress.selectorName)
             .data((d: Task, i: number) => {
                 const taskProgressPercentage = this.getDaysOffTaskProgressPercent(d);
-                // logic used for grouped tasks, when there are several bars related to one category
-                if (index === d.id) {
-                    groupedTaskIndex++;
-                } else {
-                    groupedTaskIndex = 0;
-                    index = d.id;
-                }
-                const url = `${d.id}-${groupedTaskIndex}-${window.btoa(d.taskType)}`;
                 return [{
-                    key: `${encodeURI(url)}`, values: <LinearStop[]>[
+                    key: `${encodeURI(d.url)}`, values: <LinearStop[]>[
```

Comment:
for url above you used construction like: `url(${encodeURI(`#task${task.url}`)})`);
in this line there is no **task** word. Is it expected?